supported_test.go

```diff
@@ -0,0 +1,58 @@
+package seccomp
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Could we add an entire `status` file as we do in https://github.com/containers/psgo/blob/master/internal/proc/status_test.go?

Reply:
Yep, added a whole file and the test case.